demos/serverless/template.yaml

```diff
@@ -19,6 +19,7 @@ Globals:
         MEETINGS_TABLE_NAME: !Ref Meetings
         ATTENDEES_TABLE_NAME: !Ref Attendees
         SQS_QUEUE_ARN: !GetAtt MeetingNotificationsQueue.Arn
+        BROWSER_LOG_GROUP_NAME: BrowserLogs
```

Comment:
You need to create CloudWatch log group as part of the stack

`
ChimeSDKBrowserLogs:
  Type: AWS::Logs::LogGroup
  Properties: 
    LogGroupName: BrowserLogs
`
and then refer it like this:

`
BROWSER_LOG_GROUP_NAME: !Ref ChimeSDKBrowserLogs
`